examples/mqtt-client-publish-message/mqtt_client_publish_message.bal

```diff
@@ -0,0 +1,25 @@
+import ballerina/http;
+import ballerina/mqtt;
+import ballerina/time;
+import ballerina/uuid;
+
+public type TemperatureDetails readonly & record {
+    string deviceId;
+    time:Utc timestamp;
+    decimal temperature;
+};
+
+service / on new http:Listener(9090) {
+    private final mqtt:Client temperaturePublisher;
+
+    function init() returns error? {
+        self.temperaturePublisher = check new (mqtt:DEFAULT_URL, uuid:createType1AsString());
```

Comment:
Can't we remove the uuid usage here and pass a string literal?

Reply:
Wanted to showcase the unique id creation here. WDYT @ayeshLK ? Should we remove it?

Reply:
For an ID don't we usually use a recognisable name?

Reply:
For MQTT we need to provide a unique id for each client to uniquely identify them. Otherwise, they get disconnected. That's why I needed to add UUID here. In kafka, the client id is just a label for the client.

We can remove this complicates the BBE

Reply:
Updated